src/test/java/BaseTest.java

```diff
@@ -1,13 +1,70 @@
-import org.testng.annotations.BeforeSuite;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
 
-public class BaseTest {
+import java.time.Duration;
 
+public class BaseTest {
+    String url;
+    static WebDriver driver;
     @BeforeSuite
     public static void chromeConfigs() {
-        // This is for Windows users
+
         if (System.getProperty("os.name").toLowerCase().contains("win")) {
             System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
         }
     }
+    @BeforeMethod
+    @Parameters({"baseURL"})
+    public void launchBrowser(String baseURL) {
+
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
```

Comment:
I recommend add `driver.manage().window().maximize();` (To maximize browser in Selenium, you need to call the maximize() Selenium command to maximize window interface of the driver class)